.yamato/build.yml

```diff
@@ -67,3 +67,16 @@ test_trigger:
     - .yamato/build.yml#build_for_android_win
     - .yamato/build.yml#test_on_osx
     - .yamato/build.yml#test_on_win
+
+publish_to_stevedore:
+  name: Publish (Stevedore testing)
+  agent:
+    type: Unity::VM::osx
+    image: build-system/unity-extra-macos-10.13-katana:v0.3.6-621005
+    flavor: b1.xlarge
+  commands:
+    - curl -sSo StevedoreUpload "$STEVEDORE_UPLOAD_TOOL_MAC_X64_URL"
+    - chmod +x StevedoreUpload
+    - ./StevedoreUpload --version-len=12 --repo=testing --version="$GIT_REVISION" build/*
+  dependencies:
+    - .yamato/build.yml#build_for_android_osx
```

Comment:
should we depend on tests too, since it seems it easy to upload a broken build to Stevedore?

Reply:
There's no damage in uploadin extra artifact g to testing. As long as PR doesn't land with failing tests, I think we're fine.